panther_ioc_rules/log4j_exploit_iocs.py

```diff
@@ -0,0 +1,22 @@
+from panther_analysis_tool.immutable import ImmutableList
+from panther_iocs import LOG4J_EXPLOIT_IOCS
+
+
+def nested_value_search(data, search_value):
+    if not hasattr(data, "__iter__"):
+        return False
+    if isinstance(data, ImmutableList):
```

Comment:
Is this an ImmutableList at this point or just a List?

Reply:
So this is interesting. I put a pdb trace here and the only thing that is caught as an immutable list is the[ test that has (Dict) in the title](https://github.com/panther-labs/panther-analysis/pull/357/files#diff-85b27e68e3f0b87def955de820cad3a1d59caeb59887d9561bf4fef0173a88f4R66). 
If I change this to simply `List` nothing ever matches the condition

Reply:
```python
(Pdb) isinstance("data",dict)
False
(Pdb) isinstance("data", dict)
False
(Pdb) type(data)
<class 'dict'>
```
Im very confused now